app/src/main/java/com/hedvig/app/feature/embark/ui/EmbarkActivity.kt

```diff
@@ -109,6 +109,13 @@ class EmbarkActivity : BaseActivity(R.layout.activity_embark) {
         if (offerKeys != null && offerKeys.isNotEmpty()) {
             val offerIds = offerKeys.mapNotNull { model.getFromStore(it) }
             showWebOffer(offerIds)
```

Comment:
I just check for the "Offer" passage here. Can we do this in a neater way?

Reply:
There should be an `ExternalRedirect` here, I believe, that instructs us where to go, and probably also which keys to provide as context to that external redirect location. 🤔